x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/PyTorchModelIT.java

```diff
@@ -244,10 +244,11 @@ public void testDeploymentStats() throws IOException {
             assertThat(byteSize, equalTo((int) RAW_MODEL_SIZE));
 
             Response humanResponse = client().performRequest(new Request("GET", "/_ml/trained_models/" + modelId + "/_stats?human"));
-            stats = (List<Map<String, Object>>) entityAsMap(humanResponse).get("trained_model_stats");
+            var responseMap = entityAsMap(humanResponse);
+            stats = (List<Map<String, Object>>) responseMap.get("trained_model_stats");
```

Comment:
Given the way the test is failing, it might be worth assigning this to a separate variable.